src/models/methods/subscribe.ts

```diff
@@ -76,6 +76,7 @@ export interface TransactionStream extends BaseStream {
   meta?: TransactionMetadata
   transaction: Transaction
   validated?: boolean
+  warnings?: Array<{ id: number; message: string }>
```

Comment:
Do all the streams have a warning option? Or was this added just for the partial payment stuff?

Reply:
Just for the partial payments stuff